.github/workflows/vue-todo.yml

```
@@ -0,0 +1,67 @@
+name: Example App
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - features/*
+  pull_request:
+    paths-ignore:
+      - 'webview-src/**'
+      - 'webview-dist/**'
+      - 'examples/todos-app/**'
+
```

```suggestion

env:
  CARGO_INCREMENTAL: 0
  CARGO_PROFILE_DEV_DEBUG: 0

```

incremental compilation and debuginfo aren't really helpful in CI and just bloat the target folder.